libraries/api/src/middleware/auth.js

```diff
@@ -159,14 +151,14 @@ const remoteAuthentication = ({signatureValidator, entry}) => {
     }
 
     // Send input to signatureValidator (auth server or local validator)
-    let result = await Promise.resolve(signatureValidator({
+    let result = await signatureValidator({
```

Comment:
Same here: used to be a `Promise`, and now a value. (in fact, everywhere in this file). Is that what's expected by callers?